src/components/Banner.tsx

```diff
@@ -1,7 +1,7 @@
 export const Banner = () => (
   <div className="absolute z-50 mt-20 w-full animate-gradient-xy bg-gradient-to-r from-blue-400 via-purple-500 to-teal-500 p-2 text-center text-lg font-semibold text-white">
-    <a href="/blog/windows-beta" className="hover:underline">
-      Windows support is now in beta! Learn more
+    <a href="/blog/desktop-in-production" className="hover:underline">
+      Desktop support is now production-ready!
```

Comment:
```suggestion
      Desktop support is now stable!
```

Might want to shorten to avoid ugly wrapping on mobile